### PR TITLE
Max version Dropbox disable and no sync pull animations

### LINKED ISSUE #822: Drobox sync not initiated
* **Browser**: Chrome Beta 80.0.3987.42
* **Operating System**: macOS Mojave 10.14.6
* **Stylus Version**: 1.5.6 (beta channel)
* **Screenshot**: see below

image

I want to export to Dropbox but it stays at this pop-up saying “connecting to Dropbox…”

In addition,I cannot see Stylus shown on my Dropbox account page, so I guess Stylus has issue initializing the connection.

### PR DESCRIPTION
**Pull request by narcolepticinsomniac**
Fixes #822

Max version Dropbox disable and no sync pull animations.

### CODE REVIEW

File: manage/manage.js [hunk 1]
```diff
@@ -548,7 +548,7 @@ function handleUpdate(style, {reason, method} = {}) {
     handleUpdateInstalled(entry, reason);
   }
   filterAndAppend({entry}).then(sorter.update);
-  if (!entry.matches('.hidden') && reason !== 'import') {
+  if (!entry.matches('.hidden') && reason !== 'import' && !$('#stylus-embedded-options')) {
```
Comment (eight04) [inline manage/manage.js:551]: I think we can filter out these styles with `reason !== 'sync'`:
https://github.com/openstyles/stylus/blob/c50b3cfddcf566efb5b4984f95d7b21347b3baf6/background/style-manager.js#L166

File: options/options.js [hunk 2]
```diff
@@ -9,8 +9,7 @@ setupRadioButtons();
 enforceInputRange($('#popupWidth'));
 setTimeout(splitLongTooltips);
 
-// TODO: add max version to re-enable once crbug.com/996859 is resolved
-if (!FIREFOX && CHROME >= 3809) {
+if (!FIREFOX && CHROME >= 3809 && CHROME < 4044) {
```
Comment (eight04) [inline options/options.js:12]: Let's add a ref comment:
https://github.com/openstyles/stylus/issues/822

File: manage/manage.js (hunk 1 above)
Comment (narcolepticinsomniac) [reply manage/manage.js]: Done, thanks.

File: options/options.js (hunk 2 above)
Comment (narcolepticinsomniac) [reply options/options.js:13]: Done, thanks.

**Pull request merged**